rpc/eth/api.go

```diff
@@ -332,7 +333,7 @@ func (api *PublicAPI) Call(args utils.TransactionArgs, blockNum ethrpc.BlockNumb
 	if err != nil {
 		if strings.HasPrefix(err.Error(), revertErrorPrefix) {
 			revertErr := api.NewRevertError(err)
-			api.Logger.Error("failed to execute SimulateCall, reverted", "error", err, "reason", revertErr.reason)
+			api.Logger.Error("failed to execute SimulateCall, reverted", "error", err, "reason", revertErr.Reason)
```

Comment:
(Also elsewhere, to be consistent with what we use in Oasis Core et al.)
```suggestion
			api.Logger.Error("failed to execute SimulateCall, reverted", "err", err, "reason", revertErr.Reason)
```